packages/eslint-config-modular-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-modular-app",
-  "version": "3.0.2",
+  "version": "4.0.0",
```

Comment:
This needs to go back to 3.0.2 - unfortunately after `changeset version` you have to revert all versions every time (or once at the end)

Reply:
Cleaned it up, thanks